multisig/simulations/arbitrumone/vip-019/index.ts

```diff
@@ -0,0 +1,254 @@
+      expect(await c.pendingOwner()).to.equal(ethers.constants.AddressZero);
+    });
+
+    it("owner of proxy admin is guardian", async () => {
+      expect(await proxyAdmin.owner()).to.equal(arbitrumone.GUARDIAN);
+    });
+    it("pending owner", async () => {
+      expect(await prime.pendingOwner()).to.equal(ethers.constants.AddressZero);
+      expect(await plp.pendingOwner()).to.equal(ethers.constants.AddressZero);
+    });
+
+    for (const rewardDistributor of REWARD_DISTRIBUTORS) {
+      it(`should have no pending owner for ${rewardDistributor}`, async () => {
+        const c = new ethers.Contract(rewardDistributor, OWNERSHIP_ABI, provider);
+        expect(await c.pendingOwner()).to.equal(ethers.constants.AddressZero);
+      });
+    }
+
+    it("pending owner of psr", async () => {
+      expect(await psr.pendingOwner()).to.equal(ethers.constants.AddressZero);
+    });
+
+    for (const comptrollerAddress of COMPTROLLERS) {
+      it(`should have no pending owner for ${comptrollerAddress}`, async () => {
+        const c = new ethers.Contract(comptrollerAddress, OWNERSHIP_ABI, provider);
+        expect(await c.pendingOwner()).to.equal(ethers.constants.AddressZero);
+      });
+    }
+
+    for (const vTokenAddress of VTOKENS) {
+      it(`should have no pending owner for ${vTokenAddress}`, async () => {
+        const v = new ethers.Contract(vTokenAddress, OWNERSHIP_ABI, provider);
+        expect(await v.pendingOwner()).to.equal(ethers.constants.AddressZero);
+      });
+    }
+
+    it("owner of ComptrollerBeacon is guardian", async () => {
+      expect(await comptrollerBeacon.owner()).to.equal(arbitrumone.GUARDIAN);
+    });
+
+    it("owner of VTokenBeacon is guardian", async () => {
+      expect(await vTokenBeacon.owner()).to.equal(arbitrumone.GUARDIAN);
+    });
+
+    it("pending owner of PoolRegistry", async () => {
+      expect(await poolRegistry.pendingOwner()).to.equal(ethers.constants.AddressZero);
+    });
+    it("should have no pending owner", async () => {
+      expect(await xvsVault.pendingAdmin()).to.equal(ethers.constants.AddressZero);
+    });
+
+    it("should have no pending owner", async () => {
+      expect(await xvsStore.pendingAdmin()).to.equal(ethers.constants.AddressZero);
+    });
+
+    it("should have no pending owner", async () => {
+      expect(await resilientOracle.pendingOwner()).to.equal(ethers.constants.AddressZero);
+      expect(await chainLinkOracle.pendingOwner()).to.equal(ethers.constants.AddressZero);
+      expect(await redstoneOracle.pendingOwner()).to.equal(ethers.constants.AddressZero);
+      expect(await boundValidator.pendingOwner()).to.equal(ethers.constants.AddressZero);
+    });
+  });
+
+  describe("Post-VIP behavior", async () => {
+    before(async () => {
+      await pretendExecutingVip(await vip019());
+    });
+
+    for (const converter of CONVERTERS) {
+      it(`should have no pending owner for ${converter}`, async () => {
```

Comment:
I think the title is wrong, **it should have pending owner** (remove `no`). This change should be applied to the rest of the post-vip checks